examples/SelfManagedProvider/SelfManagedProvider.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Absio.Sdk.Container;
+using Absio.Sdk.Providers;
+
+namespace SelfManagedProvider
```

Comment:
Can you add integration tests to prove this provider works?  I see your program does that to some extent.  Is that good enough?  Probably have them all hit the sandbox server since this is public facing.  Leave a not about the API key being tied to this sample application (you should probably get one from Scott).